tests/Unit/Framework/ActionTesting.hpp

```diff
@@ -445,8 +445,6 @@ struct ForwardAllOptionsToDataBox<tmpl::list<SimpleTags...>> {
 };
 }  // namespace detail
 
-/// \cond
```

Comment:
no endcond to remove?

Reply:
no, there were more conds than endconds...

Reply:
Okay, thanks!